docs/tutorial-node-implementation.md

```diff
@@ -0,0 +1,462 @@
+# Tutorial: Create Custom Nodes
+
+This tutorial takes you through the steps to create a new node with implementation to use it in a TOSCA topology.
+The tutorial will lead you through recreating a node from the [Tutorial: Model and Deploy an Application with OpenTOSCA](./tutorial-model-and-deploy.md).
+
+:information_source: Terms in *italics* can be found in the [glossary](tosca-glossary.md).
+
+**Requirements:**
+
+ *  [GIT](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) (with GIT-LFS)
+ *  [Docker](https://docs.docker.com/get-docker/) & [Docker Compose](https://docs.docker.com/compose/install/)
+ *  Optional: At least 6GB RAM free for the OpenTOSCA complete docker-compose
+
+Before starting with this tutorial clone the <https://github.com/OpenTOSCA/opentosca-docker> repository and read through the relevant sections of the README once (e.g. the first two sections, tips and tricks and troubleshooting).
```

Comment:
Before you start with...
After tutorial, add a comma.
Add a comma before `and` when you do a list in English.